stdlib/universe/fill.go

```diff
@@ -264,5 +278,23 @@ func (t *fillTransformation) fillTable(w *table.StreamWriter, cr flux.ColReader,
 		}
 		vs[i] = t.fillColumn(col.Type, arr, fillValue)
 	}
+
+	// If the fill column is new, create a completely null column and call fillColoumn on it
+	if wColsLen > crColsLen {
```

Comment:
With the change above, instead of this condition, you can now do:

```
if vs[colIdx] == nil {
    // fill in a bunch of nil values
}
```

I think this is more straightforward than checking the number of columns and comparing them.